google/oauth2/service_account.py

```diff
@@ -556,6 +543,11 @@ def __init__(
             additional_claims (Mapping[str, str]): Any additional claims for
                 the JWT assertion used in the authorization grant.
             quota_project_id (Optional[str]): The project ID used for quota and billing.
+            universe_domain (str): The universe domain. The default
+                universe domain is googleapis.com. If this value is the default
```

Comment:
Same suggestion to maybe simplify: "If this value is the default value, then" -> "For default value" or such

Reply:
done